static-src/scripts/maps/state/constants.js

```diff
@@ -19,4 +19,5 @@ export const TILE_LAYER_TYPE = 'tiles'
 *   Boundaries of San Francisco for map defaults.
 */
 
-export const SAN_FRANCISCO_BOUNDS = [ [ 37.813996, -122.529439 ], [ 37.702302, -122.348852 ] ]
```

Comment:
These changes shouldn't be in the PR.  It looks like you might have created this branch while on your last feature branch?  The `git branch new-branch-name` and `git checkout -b new-branch-name` commands will create a branch based on whatever branch you're currently on, so you'll need to switch back to your `master` before creating a new branch.  Do you mind deleting and recreating this branch with the same changes but based on `master`?